src/search/FindInFiles.js

```diff
@@ -73,12 +73,20 @@ define(function (require, exports, module) {
     var searchModel = new SearchModel();
 
     /* Forward declarations */
-    var _documentChangeHandler, _fileSystemChangeHandler, _fileNameChangeHandler, clearSearch;
+    var _documentChangeHandler,
+        _fileSystemChangeHandler,
+        _processCachedFileSystemEvents,
+        _debouncedFileSystemChangeHandler,
+        _fileNameChangeHandler,
+        clearSearch;
+    
+    /* Waits for FS changes to stack up until processing them (scripts like npm install can do a lot of movements on the disk) */
+    var FILE_SYSTEM_EVENT_DEBOUNCE_TIME = 1000;
```

Comment:
You're right here, re-tested this set to 100ms with my script and it's fine. So setting it to 100ms.